repository/src/main/java/org/apache/atlas/repository/store/aliasstore/ESAliasStore.java

```diff
@@ -212,7 +213,10 @@ private void personaPolicyToESDslClauses(List<AtlasEntity> policies,
                 } else if (getPolicyActions(policy).contains(ACCESS_READ_PERSONA_DOMAIN)) {
 
                     for (String asset : assets) {
-                        terms.add(asset);
+                        asset = validateAndConvertAsset(asset);
```

Comment:
instead of this, create a method that will return whether the asset is all domains or not, that method will check all 3 values ie NEW_WILDCARD_DOMAIN_SUPER, */super, *

Reply:
Have made changes, please check if what i have changed is what you meant to say.